### ISSUE TITLE
Bugs in Local preferences in cooperation with screen readers

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#1122]
**Reported by krzyszp on 30 Sep 41858557 05:29 UTC**
In Local preferences (advanced view) labels for text fields are not readable by NVDA (screen reader). (Manager 6.10.13)

**Comment by romw** [comment]
**Commented by Ageless on 13 Mar 41859065 06:23 UTC**
Duplicate of #1150
